picoquic/picoquic_internal.h

```diff
@@ -614,6 +614,7 @@ typedef struct st_picoquic_cnx_t {
     unsigned int client_mode : 1; /* Is this connection the client side? */
     unsigned int key_phase_enc : 1; /* Key phase used in outgoing packets */
     unsigned int key_phase_dec : 1; /* Key phase expected in incoming packets */
+    unsigned int data_acknowledged : 1; /* 1RTT data acknowledged by peer */
```

Comment:
Maybe should be renamed to `data_1rtt_acknowledged`.

Reply:
OK. Renamed to "one_rtt_data_acknowledged".